class/Helper.js

```diff
@@ -12,5 +12,12 @@ module.exports = {
             obj.email = req.body.email
             
         req.flash('form', obj);
+    },
+
+    isEmail : (value) => {
```

Comment:
Why put this in a class ?
You know a file can export several function.
No neet to group several unrelated function in a class